scripts/variantstore/wdl/extract/generate_ordered_paths.py

```diff
@@ -0,0 +1,17 @@
+import argparse
+
+def generate_ordered_paths(root_path, path_suffix, number):
+    for i in range(0, number):
+        print(f"{root_path}{i}{path_suffix}")
```

Comment:
do these need to be zero-padded or anything?

Reply:
nope, see https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/wdl/GvsExtractCallset.wdl#L92